Chargebee/Classes/Cache/CBCache.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  CBCache.swift
+//  Chargebee
+//
+//Created by ramesh_g on 28/07/23.
+//
+
+import Foundation
+
+
+struct ConfigCacheModel: Codable {
+    var config: CBAuthentication
+    var date: Date
```

Comment:
Can we name this better? Is this the entry created time or the time till the entry is valid?

Reply:
> Can we name this better? Is this the entry created time or the time till the entry is valid?

updated